addons/mail/static/src/discuss/call/common/call_participant_card.scss

```diff
@@ -61,3 +69,18 @@
         background-color: $o-gray-800;
     }
 }
+
+.o-discuss-Call-insetCard {
```

Comment:
It should have a higher z-index (or different stacking context) than the `o-discuss-CallParticipantCard-overlay` which has the `z-index-1` class. Currently the name of cards that are below the inset card are on top of it.

Reply:
Small issue that can be fixed in follow-up PR.
It's pretty easy to see the UI overlap, so we'll eventually fix it from a ticket and/or by actively using the feature